LibSWBF2/Wrappers/CollisionMesh.h

```diff
@@ -25,14 +25,14 @@ namespace LibSWBF2::Wrappers
 		CollisionMesh(); 
 
 		coll *p_CollisionChunk = nullptr;
-		List<uint32_t> m_Indicies;
+		mutable List<uint32_t> m_Indicies;
 
```

Comment:
Allows modification from ```const``` methods, thus we can keep the pre-merge public API for ```Model``` etc